docker-compose.yml

```diff
@@ -0,0 +1,174 @@
+version: '3.7'
+services:
+  provisioning-db:
+    restart: on-failure
+    image: docker.io/postgres:13.1
+    healthcheck:
+      test: ["CMD", "pg_isready", "-U", "postgres"]
+      interval: 2s
+      timeout: 3s
+      retries: 10
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+      - POSTGRES_DB=provisioning
+    volumes:
+      - ./compose-data/postgres/backend:/var/lib/postgresql/data
+    expose:
+      - 5432
```

Comment:
You name this "provisioning-db" and dependnency is named "backend". I suggest to pick be consistent, pick one or the other, for example "backend" and "backend-db".

Also can you give this the "backend" tag? It will be very useful to have tags for every single service, I will be likely using this to run just sources or sometimes just notifications etc.